packages/sdks/e2e/tests/src/tests/main.spec.ts

```diff
@@ -75,13 +75,13 @@ test.describe(targetContext.name, () => {
 
     await expect(
       await getElementStyleValue({
-        locator: page.locator(`text="This text should be red"`).first(),
+        locator: page.locator(`.builder-text`).first(),
```

Comment:
This might not work in react-native, since react native does not have CSS classes. Selection by text will be more accurate

Reply:
But this is what fails the test in Solid.

Reply:
weird that this would be an issue, but regardless: if you need to do something just for react native, this is how:

```ts
const isRNSDK = process.env.SDK === 'reactNative';
```

and then check things in a different way for React Native.